simtools/simulator.py

```diff
@@ -0,0 +1,740 @@
+
+        return self._results["output"]
+
+    def getListOfHistogramFiles(self):
+        """
+        Get list of histogram files.
+        (not applicable to all simulation types)
+
+        Returns
+        -------
+        list
+            List with the full path of all the histogram files.
+        """
+        self._logger.info("Getting list of histogram files")
+        return self._results["hist"]
+
+    def getListOfInputFiles(self):
+        """
+        Get list of input files.
+
+        Returns
+        -------
+        list
+            List with the full path of all the input files.
+        """
+        self._logger.info("Getting list of input files")
+        return self._results["input"]
+
+    def getListOfLogFiles(self):
+        """
+        Get list of log files.
+
+        Returns
+        -------
+        list
+            List with the full path of all the log files.
+        """
+        self._logger.info("Getting list of log files")
+        return self._results["log"]
+
+    def printListOfOutputFiles(self):
+        """Print list of output files."""
+        self._logger.info("Printing list of output files")
+        self._printListOfFiles(which="output")
+
+    def printListOfHistogramFiles(self):
+        """Print list of histogram files."""
+        self._logger.info("Printing list of histogram files")
+        self._printListOfFiles(which="hist")
+
+    def printListOfInputFiles(self):
+        """Print list of output files."""
+        self._logger.info("Printing list of input files")
+        self._printListOfFiles(which="input")
+
+    def printListOfLogFiles(self):
+        """Print list of log files."""
+        self._logger.info("Printing list of log files")
+        self._printListOfFiles(which="log")
+
+    def _makeResourcesReport(self, inputFileList):
+        """
+        Prepare a simple report on computing resources used
+        (includes run time per run only at this point)
+
+        Parameters
+        ----------
+        inputFileList: str or list of str
+            Single file or list of files of shower simulations.
+
+        Returns
+        -------
+        dict
+           Dictionary with reports on computing resources
+
+        """
+
+        if len(self._results["sub_out"]) == 0 and inputFileList is not None:
+            self._fillResultsWithoutRun(inputFileList)
+
+        runtime = list()
+        nEvents = None
+
+        for run in self.runs:
+            nEvents, thisRuntime = self._simulationRunner.getResources(
+                runNumber=run)
+            if thisRuntime:
+                runtime.append(thisRuntime)
+
+        meanRuntime = np.mean(runtime)
+
+        resources = dict()
+        resources["Runtime/run [sec]"] = meanRuntime
+        if nEvents and nEvents > 0:
+            resources["#events/run"] = nEvents
+            resources["Runtime/1000 events [sec]"] = meanRuntime * 1000 / nEvents
```

Comment:
You should probably mention here that this is wall time

Reply:
Clarified docstring.